### PR TITLE
CSE-143 Update comments in ClientApi.purs, added delete walletset.

### PR DESCRIPTION
**Pull request by ksaric**
https://issues.serokell.io/issue/CSE-143

Also added `deleteWalletSet` endpoint from https://github.com/input-output-hk/cardano-sl/pull/365.

### CODE REVIEW

**Comment by Martoon-00** [comment]
Oh, @akegalj I'm so sorry, looks like I didn't merge CSM-261 into CSM-258 fully, and thus we lost `deleteWalletSet` 🤦‍

**Comment by ksaric** [comment]
That's ok @Martoon-00 - it's here, it's not lost :smiley: 
I just copied it anyway.

File: daedalus/src/Daedalus/ClientApi.purs [hunk 1]
```diff
@@ -131,13 +162,18 @@ renameWalletSet = mkEffFn2 \wSetId name -> fromAff <<< map encodeJson $ B.rename
...
+-- |   cwsWSetMeta:
+-- |    { cwsUnit: 0,
+-- |      cwsName: 'Genesis wallet set',
+-- |      cwsAssurance: 'CWANormal' },
+-- |   cwsPassphraseLU: 1495545014.377285,
+-- |   cwsId: '1feqWtoyaxFyvKQFWo46vHSc7urynGaRELQE62T74Y3RBs8',
 -- |   cwsHasPassphrase: false,
--- |   cwsId: '1gCC3J43QAZo3fZiUTuyfYyT8sydFJHdhPnFFmckXL7mV3f' }
+-- |   cwsAmount: { getCCoin: '0' } }
+-- | ```js
```
Comment (Martoon-00) [inline daedalus/src/Daedalus/ClientApi.purs]: I'm not an expert, but in docs to other methods there is no `'''js` at the end

**Comment by Martoon-00** [comment]
😥

File: daedalus/src/Daedalus/ClientApi.purs (hunk 1 above)
Comment (ksaric) [inline daedalus/src/Daedalus/ClientApi.purs]: Nice catch!

**Pull request merged**